### DISCUSSION TITLE
Has anyone ever tried do some drag & drop ?

### DISCUSSION THREAD

**Discussion by 2ant** [Bodmer/TFT_eSPI#2313] [General]
Hi, im curious to know if someone tried to make some drag & drop functionnality with or without tft_espi, or if anyone know of some ressources.

I managed to do this : 

https://user-images.githubusercontent.com/52797081/211567928-1c8ac72f-f9cf-440d-b96a-e6559c5c3a82.mp4

The end goal will probably be some rpg inventory.

**Comment by TangerineDreamer** [comment]
Felicitations,did you find a new way to build a gui or a menu?

**Reply by 2ant** [reply]
Thx, i'm just trying different things with tft_espi to learn it and later do some cool projects.

**Comment by TangerineDreamer** [comment]
It looks like amazing,is it a bodmer example?

**Reply by 2ant** [reply]
Nope, just me trying some stuff, if i manage to get rid of the flicker and clean up the code i'll share it.
